source/standards/storing-source-code/index.html.md.erb

```diff
@@ -8,33 +8,160 @@ old_paths:
+  very visible to other users
+
+## Git Repository Hosting - GitHub and Azure DevOps (ADO)
+
+While not required, most git users will nominate one copy of the git repository to be the authoritative copy.
+
+- It is possible to self-host a git server for this purpose but, often, this will be a hosted solution such as
+  GitHub, Azure DevOps, GitLab, or any of the numerous other commercial services available.
+- A "hub and spoke" is easier to reason about and keep synchronised
+- Integrations with other tools will work with less friction, where they have a single copy to work with
+  (e.g., automated test/deployment tools, issue/bug management)
+
+### GitHub
+
+Historically, some projects use (and may remain on) private Azure DevOps and/or private GitHub repositories
+for legacy reasons, though we are now required
+to [make new source code open](https://apply-the-service-standard.education.gov.uk/service-standard/12-make-new-source-code-open.html).
+
+Specifically, we use GitHub for new and migrated work.
+
+### GitHub Organisations
+
+Department for Education (DfE) source code repositories on GitHub should be stored under an appropriate
+organisation, thereby giving appropriate oversight and protections to these source code repositories.
+
+Specifically:
+
+- The [Department for Education Digital](https://github.com/DfE-Digital)
+  GitHub organisation is used for new and existing source code repositories
+  - This is applicable to production and prototype code
+- Work created outside the DfE Digital organisation should be transferred into
+  the DfE Digital organisation at the earliest opportunity.
+  - [GitHub: Guide to transferring a repository](https://help.github.com/en/articles/transferring-a-repository)
+- The [Skills Funding Agency](https://github.com/SkillsFundingAgency/)
+  GitHub organisation is also used by the DfE
+  - Not the default for DfE-Digital
+
+If your account is added to a repository without the account being a member of the owning organisation,
+it will be counted and labelled as
+an ["outside collaborator"](https://docs.github.com/en/organizations/managing-user-access-to-your-organizations-repositories/adding-outside-collaborators-to-repositories-in-your-organization).
+
+To join a GitHub organisation, follow the guidance and request forms available
+via [Digital Tools Support](<%= data.site.digital_tools %>).
+As there is a small cost implication for accounts to be added to a GitHub organisation,
+this should normally be done via / with support from your Delivery Manager.
+
+### Work vs Personal GitHub accounts
+
+You may use your personal GitHub account, but you should:
+
+- [GitHub: Add your DfE email address to your account](https://help.github.com/articles/adding-an-email-address-to-your-github-account/)
+- [GitHub: Use your secondary (DfE) email address for notifications](https://help.github.com/articles/managing-notification-emails-for-organizations/)
```

Comment:
I think using personal accounts is pretty much the default rather than something that's allowed. Also not sure we need to be telling people which email addresses to add to their account or how they should receive notifications. Feels a bit prescriptive and I can't really see any benefit.

Reply:
Sounds like we have variation across DfE :) 

In my local bubble when this got discussed during some folks onboarding, I think the feeling was that work-related notifications wouldn't appear on work devices unless you add your work email address so this became the unofficial suggestion at the time.

Reflecting on it more now, it seems stranger that this would mean that non-work notifications would appear in work inboxes and my personal previously-on-the-fence position has now shifted against doing this.

Reply:
But yes, I agree that this should likely be removed or restructured/rephrased.

If we do have any semblance of an official position (e.g., if we encourage use of personal GitHub accounts) then that would be good to include.

Reply:
I hadn't really thought of it from a notifications point of view, I disable email notifications and rely on GitHub's blue bell.

If you take a browse through the [organisation's people list](https://github.com/orgs/DFE-Digital/people) there are a few accounts containing 'DfE' here and there but the vast majority look like personal ones.

Based on my initial flick through there's a split between digital/non-digital folks, so maybe the guidance should be given at that level.